src/oatpp/core/data/mapping/type/Primitive.hpp

```diff
@@ -298,7 +298,7 @@ class Boolean : public type::ObjectWrapper<bool, __class::Boolean>  {
     return !operator == (other);
   }
 
-  inline operator bool() const {
+  explicit inline operator bool() const override {
```

Comment:
Please do not make it `explicit`.
See the `oatpp::Boolean` conversion table in the [comment](https://github.com/oatpp/oatpp/pull/274#issuecomment-658173567)